ArmTemplates/Application-Insights/metric-alerts.json

```diff
@@ -115,10 +122,11 @@
                         {
                             "name": "metric critria",
                             "metricName": "[parameters('metricName')]",
-                            "dimensions": [],
+                            "dimensions": "[parameters('dimensions')]",
                             "operator": "[parameters('operator')]",
                             "threshold": "[parameters('threshold')]",
-                            "timeAggregation": "[parameters('aggregation')]"
+                            "timeAggregation": "[parameters('aggregation')]",
+                            "criterionType": "StaticThresholdCriterion"

```

Comment:
This is a required property according to the schema,  and currently we only use static thresholds...